app/controllers/organisations_controller.rb

```diff
@@ -155,6 +155,7 @@ def update
         end
         redirect_to details_organisation_path(@organisation)
       else
+        @used_rent_periods = @organisation.lettings_logs.pluck(:period).uniq.compact.map(&:to_s)
```

Comment:
This is irrelevant, it's failing now if we try to update the org details to something invalid (like an empty name)